components/calendar/demo/week-number.md

```diff
@@ -0,0 +1,7 @@
+## zh-CN
+
+通过将 `showWeekNumber` 属性设置为 `true`，在全屏日历中显示周数。
```

Comment:
showWeekNumber => showWeek

Reply:
Yes, DatePicker use `showWeek` as prop name: https://ant.design/components/date-picker

Reply:
Ok, I have renamed the prop as requested. Can you check now if the PR can be merged? Thanks.